### ISSUE TITLE
Step by step installation and configuration of the API when the WebStore does not allow downloading it.

### ISSUE THREAD

**Issue by Lewisihno** [yotam180/WhatsappAPI#44]
step by step installation and configuration of the API when the WebStore does not allow downloading, because I access the link to install the extension but says there is an error in the link, look for it but it does not appear, I would like to launch this Chatbot for some Labs that I have in mind: D

**Comment by rcarmonad** [comment]
Here is the answer for your question
https://github.com/yotam180/WhatsappAPI/issues/38

**Comment by Lewisihno** [comment]
Thanks. 
https://github.com/yotam180/WhatsappAPI/issues/38

now, to try!!

**Comment by rcarmonad** [comment]
@Lewisihno i've got an exception: Uncaught ReferenceError: Store is not defined.
Is this working for you?

I'm using chrome on ubuntu.

**Comment by yotam180** [comment]
@rcarmonad Let me update the code there and test it again. I will notify here when done :slightly_smiling_face:

**Comment by yotam180** [comment]
@rcarmonad I think I got it. Try again and keep me updated.

**Comment by rcarmonad** [comment]
Ok, i've been working on it and see that the `main.js` was more updated than the files in the `ext` folder.
I'll keep you updated.

**Comment by yotam180** [comment]
Yes it was. Try updating your git folder, the `main.js` file is now gone and I migrated all the changes and fixes to the extension `api.js`.

**Comment by rcarmonad** [comment]
There are some issues with changes on the json from whatsapp.
For example:

```
parseMsgObject: function(msg_object) {
			var m = msg_object.all;
			if (msg_object["__x__quotedMsgObj"]) {
				m.quotedMsg = API.parseMsgObject(Core.msg(msg_object.__x__quotedMsgObj.__x_id._serialized));
			}
			m.chat = m.chat.all;
			delete m.msgChunk;
			return m;
		},
```
`m.chat.all` does not works. So, i've eliminated the call to `parseMsgObject`.

Previously, was other issues, with de `window.Store`, but, it was solved on `main.js`, and now in the extension.
Let me know if i can help you in any way.
